.github/actions/nf-test-action/action.yml

```diff
@@ -35,7 +35,7 @@ runs:
     - name: Install nf-test
       uses: nf-core/setup-nf-test@v1
       with:
-        version: "${{ env.NFTEST_VERSION }}"
+        version: "${{ env.NFT_VER }}"
```

Comment:
https://github.com/nf-core/modules/pull/7019

Is `NFT_VER` special? `NXF_VER` as well I couldn't remember in that PR but renovate wanted `_VERSION` by default.

Reply:
NXF_VER is nextlfow standard: https://www.nextflow.io/docs/latest/install.html#updates